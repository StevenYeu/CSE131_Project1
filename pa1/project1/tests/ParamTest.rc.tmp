Error, "project1/tests/ParamTest.rc": 
  Argument passed to reference parameter x (type int) is not a modifiable L-value.
Error, "project1/tests/ParamTest.rc": 
  Argument of type float not assignable to value parameter y, of type int.
Compile: failure.
