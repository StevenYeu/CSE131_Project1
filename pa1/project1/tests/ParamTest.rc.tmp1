  Argument passed to reference parameter x (type int) is not a modifiable L-value.
  Argument of type float not assignable to value parameter y, of type int.
Compile: failure.
