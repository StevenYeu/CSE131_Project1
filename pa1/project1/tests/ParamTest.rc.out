Error, "/tmp/cs131_testrunner_JJ0qMF/file.rc": 
  Argument passed to reference parameter x (type int) is not a modifiable L-value.
Error, "/tmp/cs131_testrunner_JJ0qMF/file.rc": 
  Argument of type float not assignable to value parameter y, of type int.
Compile: failure.
